artemis-web/src/main/java/org/apache/activemq/artemis/component/WebServerComponent.java

```diff
@@ -219,6 +239,18 @@ public synchronized void start() throws Exception {
       cleanupTmp();
       server.start();
 
+      /*
```

Comment:
I would revert this change. 
the metrics call to jmx which is instrumented to audit, with out this filter the audit on metrics will be less informative.
in addition, there are cases where access to metrics will need authentication, especially when there is RBAC on JMX mbeans.